admin/test/test_packaging.py

```diff
@@ -41,6 +42,9 @@
 require_dpkg = skipIf(not which('dpkg'), "Tests require the ``dpkg`` command.")
 require_lintian = skipIf(not which('lintian'),
                          "Tests require the ``lintian`` command.")
+require_not_ubuntu = skipIf(
+    platform.linux_distribution()[0] == 'Ubuntu',
+    "rpmlint returns spurious results on Ubuntu: FLOC-3564.")
```

Comment:
It would be exactly equivalent to make this "require centos" since we only have two supported platforms.  I would have spelled it that way ... but since it's exactly equivalent I guess I shouldn't say anything.